packages/devtools_app/test/test_infra/goldens/inspector_service_details_tree.txt

```diff
@@ -12,6 +12,7 @@ MaterialApp
          └─Semantics
            │ container: false
            │ properties: SemanticsProperties
+           │ identifier: null
            │ tooltip: null
```

Comment:
Don't these need to be removed now?

Reply:
That's right.

I was waiting for framework roll.